### DISCUSSION TITLE
Please tell me your favorite sweets.

### DISCUSSION THREAD

**Discussion by kanariari3** [NAIST-SE/AttractingContributors#1063] [2024 Attraction Contributions]
My hobby is cooking sweets. 
Do you like sweets?
So please tell me your favorite sweets.
I want to make it when I feel like it. 

Please edit here and send me your pull requests⤵
Click: https://github.com/kanariari3/AttractingContributors

**Comment by 37integer** [comment]
cake

**Reply by kanariari3** [reply]
Thank you, send your comment !
I like it too.

**Comment by natsu9731** [comment]
pudding

**Reply by kanariari3** [reply]
Thank you for your comment.
I 'm happy to know your favorite sweets.

**Comment by yuruihito** [comment]
Mont Blanc

**Reply by kanariari3** [reply]
Thank you for your participation.
That's nice answer.

**Comment by gill876** [comment]
Mine is mochi!

Please make a contribution to my project!
https://github.com/NAIST-SE/AttractingContributors/discussions/1089#discussion-6846489

**Reply by kanariari3** [reply]
Thank you for your answer.
That is woderful Japanese sweets.

Thank you for inviting me to your discussion.
I was happy to hear that. 
I answered my favorite hiking place in your repository.
